contracts/crowdsale/Crowdsale.sol

```diff
@@ -140,8 +137,8 @@ contract Crowdsale is Ownable, State {
   // WE SHOULD APPLY TAXES HERE
   function payback() public payable isActiveState {
 
-    // send payback to contract
-    address(this).transfer(msg.value);
+    // probably, we want to check if the payback agrees with 
+    // an amount, specified in the ISA
```

Comment:
Probably best to allow arbitrary payments and let backend check at the deadline of each payment whether enough has been deposited by the student through this function?

Reply:
Fine by me! I'll delete that comment 👍